File: BackEnd/ELearningAPI/ELearningAPI/TMSBO/Model/UserTopic.cs
using Library;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace TMSBO.Model
{
    [ConfigTable("user_topic", "", false, "")]
    public class UserTopic : BaseModel
    {
        [Key]
        /// <summary>
        /// user ID
        /// </summary>
        public int ID { get; set; }
        /// <summary>
        /// 
        /// </summary>
        public int? UserID { get; set; }
        /// <summary>
        /// tên user
        /// </summary>
        public string UserName { get; set; }
        /// <summary>
        /// bài học id
        /// </summary>
        public int? TopicID { get; set; }
        /// <summary>
        /// tên bài học user đang học
        /// </summary>
        public string TopicName { get; set; }
        /// <summary>
        /// 
        /// </summary>
        public int? TopicStatus { get; set; }
        /// <summary>
        /// ngày bắt đầu học
        /// </summary>
        public string StartDate { get; set; }
        /// <summary>
        /// thời gian hoàn thành
        /// </summary>
        public string FinishDate { get; set; }
        /// <summary>
        /// 
        /// </summary>
        public int? TennantID { get; set; }
    }
}

File: FrontEnd/client-app/src/app/shared/model/User_Topic.ts
/**
* Model cho user_topic để lưu lên server
 * @export
* @class EventForInsert
*/
export class UserTopic 
{
// user ID
ID:  number 
// 
UserID?:  number 
// tên user
UserName:  string 
// bài học id
TopicID?:  number 
// tên bài học user đang học
TopicName:  string 
// 
TopicStatus?:  number 
// ngày bắt đầu học
StartDate:  string 
// thời gian hoàn thành
FinishDate:  string 
// 
TennantID?:  number 
}
